rocAL/include/pipeline/context.h

```diff
@@ -27,17 +27,21 @@ THE SOFTWARE.
 #include "pipeline/master_graph.h"
 
 struct Context {
-    explicit Context(size_t batch_size, RocalAffinity affinity, int gpu_id, size_t cpu_thread_count, size_t prefetch_queue_depth, RocalTensorDataType output_tensor_type) : affinity(affinity),
-                                                                                                                                                                            _user_batch_size(batch_size) {
+    explicit Context(size_t batch_size, RocalAffinity affinity, int gpu_id, size_t cpu_thread_count, size_t prefetch_queue_depth, RocalTensorDataType output_tensor_type, RocalBatchPolicy last_batch_policy, bool last_batch_padded) : affinity(affinity),
+                                                                                                                                                                                                                                        last_batch_policy(last_batch_policy),
```

Comment:
Do we need to set these 2 variables in the context, can we not directly pass to the master graph, like cpu_thread_count?

Reply:
The cpu thread count is also passed from the pipeline -> during context creation

Reply:
We are doing the same for cpu_thread_count

Reply:
I meant initializing the variable, the two variables last_batch_policy and last_batch_padded are unused in this Context file, so why do we need to introduced private variables for these.
You can still have arguments in context, but can we not initialize the variables and pass directly like cpu_thread_count is what I meant. Also check if google coding guidelines mentions anything about this.

Reply:
Removed the internal variables